scipy/stats/_stats_py.py

```diff
@@ -9042,14 +9051,58 @@ def wasserstein_distance(u_values, v_values, u_weights=None, v_weights=None):
     :math:`\mathbb{R} \times \mathbb{R}` whose marginals are :math:`u` and
     :math:`v` on the first and second factors respectively.
 
-    If :math:`U` and :math:`V` are the respective CDFs of :math:`u` and
-    :math:`v`, this distance also equals to:
+    In the 1-dimensional case, let :math:`U` and :math:`V` denote the
+    respective CDFs of :math:`u` and :math:`v`, this distance also equals to:
 
     .. math::
 
         l_1(u, v) = \int_{-\infty}^{+\infty} |U-V|
 
-    See [2]_ for a proof of the equivalence of both definitions.
+    See [3]_ for a proof of the equivalence of both definitions.
+
+    In the more general(higher dimensional) and discrete case, it is also
+    called the optimal transport problem or the Monge problem, which can be
+    expressed in this form,
+
+    Given :math:`\Gamma` the transport plan, :math:`D` the distance matrix and
```

Comment:
This doesn't seem to be rendering:

![image](https://user-images.githubusercontent.com/6570539/204123357-18337122-b40e-41fe-8cb2-7fe6bbdd929a.png)

You can see the rendered docs by clicking:
![image](https://user-images.githubusercontent.com/6570539/204123378-d25569c2-50dd-47df-8a81-d75562b0de12.png)

Reply:
Better!

Reply:
Thanks Matt :rofl: !